lib/generate-id.js

```diff
@@ -21,12 +20,47 @@ const findId = async ( model, length ) => {
   return id
 }
 
+const idTypes = {
+  shabad: [ Shabads, 3 ],
+  line: [ Lines, 4 ],
+}
+
 /**
- * Generates a line id and a shabad id, using the database.
- */
+  * Logs the id to the console
+  * @param {Number} count the count of id to generate
+  * @param {String} type check `availableTypes`
+  */
+const logGenerateIds = async ( count, type ) => {
+  const newId = idTypes[ type ]
+
+  // Grab ids to exclude from database
+  const ids = new Set( ( await newId[ 0 ].query().select( 'id' ) ).map( ( { id } ) => id ) )
+
+  // Loop to print id's
+  for ( let i = 0; i < count; i += 1 ) {
+    console.log( `${await findId( ids, newId[ 1 ] )}` )
```

Comment:
`newId[ 1 ] ` is a bit obfuscated in meaning. I can tell you're passing in the length of the ID from reading all the code, but not from solely scanning that line.

Instead, destructure it before the loop (along with line 37's use of `newId[ 0 ]`):

`const [ Model, idLength ] = newId`